core/src/main/java/com/sequenceiq/cloudbreak/controller/mapper/SpringAccessDeniedExceptionMapper.java

```diff
@@ -1,15 +0,0 @@
-package com.sequenceiq.cloudbreak.controller.mapper;
-
-import javax.ws.rs.core.Response.Status;
-import javax.ws.rs.ext.Provider;
-
-import org.springframework.security.access.AccessDeniedException;
-
-@Provider
-public class SpringAccessDeniedExceptionMapper extends BaseExceptionMapper<AccessDeniedException> {
```

Comment:
Why deleting this class and SpringBadRequestExceptionMapper? If someone throws this exception, the response status will not be defined by us anymore.

Reply:
You are right, thanks